### PR TITLE
Tutorial tam tam

### LINKED ISSUE #426: Add a tutorial to the TamTam Micro activity
Add a tutorial to the MazeWeb activity following tutorial 9 in Sugarizer development tutorial: https://github.com/llaske/sugarizer/blob/dev/docs/tutorial_step9.md

The tutorial:
* [ ] Should display a welcome message to explain the activity (could reuse the help message describing the activity in Sugarizer activities List view)
* [ ] Should be localized
* [ ] Should respect the Sugarizer UI, as explained in tutorial

### PR DESCRIPTION
**Pull request by Larissagilliane**
Add Tutorial to Tam Tam Micro activity, like said at issue #426 . 
Closes #426 

@llaske

### CODE REVIEW

**Comment by llaske** [comment]
Nice.
Two remarks:
- Your editor seems to reformat whitespaces into originals files  (index.html, activity.js, webl10n.js). Could you limit change in files only to lines when some update is need?
- There is no sub domain in localization into Sugarizer. So - sorry for that :wink: - but PT-BR should be just PT

File: activities/TamTamMicro.activity/index.html [hunk 1]
```diff
@@ -2,43 +2,42 @@
 <html>
 
 <head>
-	<meta charset="utf-8" />
-	<title>TamTam Micro Activity</title>
-	<meta name="viewport"
-		content="user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1, width=device-width, viewport-fit=cover" />
+<meta charset="utf-8" />
```
Comment (llaske) [inline activities/TamTamMicro.activity/index.html]: Do not change indentation in file.

File: activities/TamTamMicro.activity/js/activity.js [hunk 2]
```diff
@@ -8,7 +8,6 @@ define(["sugar-web/activity/activity", "tutorial"], function (activity, tutorial
 		tutorial.start();
 	});
 
-
```
Comment (llaske) [inline activities/TamTamMicro.activity/js/activity.js]: Do not change indentation in file.

File: activities/TamTamMicro.activity/js/activity.js [hunk 3]
```diff
@@ -19,15 +18,12 @@ define(["sugar-web/activity/activity", "tutorial"], function (activity, tutorial
 		sound.renderInto(document.getElementById("audio"));
 
 		// Launch main screen
-		app = new TamTam.App({ activity: activity });
+		app = new TamTam.App({activity: activity});
```
Comment (llaske) [inline activities/TamTamMicro.activity/js/activity.js]: Do not change indentation in file.

**Comment by llaske** [comment]
Sorry, there is still lot of indentation issues to fix. I can't merge it.
It's important for traceability and maintenance reason to update only lines that need change and to respect the original indentation.

**Comment by Larissagilliane** [comment]
@llaske as soon as possible time I will make the adjustments.

**Comment by llaske** [comment]
Fixed in https://github.com/llaske/sugarizer/pull/506

**Pull request closed without merging**